.env.example

```diff
@@ -1,14 +1,14 @@
-NODE_ENV=development
-DB_USERNAME=timitejumola
-DB_PASSWORD=Oluwatimileyin9595
-DB_DATABASE=errorswag
-DB_HOST=localhost
+NODE_ENV=
```

Comment:
Good job 👍  @timi-codes,
I noticed you removed the details contained in the `.env.example` file to maintain security in the application.

However, the PR description is not good enough to guide a user on how to test the feature implemented. Hence there is room for improvement. 
Going forward, you can look at the link [here](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices) for some insights.

There are some reviews below.